CHANGES.rst

```diff
@@ -4,12 +4,22 @@ Release 7.4.0 (in development)
 Dependencies
 ------------
 
+* LaTeX: the ``xcolor`` package is now required.  It is anyhow part of
+  Ubuntu ``texlive-latex-recommended`` which has always been required.
+
+* LaTeX: the ``fontawesome5`` package is needed for the default choice of
+  icons in admonition titles (alternative user configuration is possible via
+  the suitable :ref:`'sphinxsetup' <latexsphinxsetup>` keys).
+
```

Comment:
I probably need to make clearer here that in absence of `fontawesome5` nothing bad happens: the PDF will display no icon, that's all.  And user can use the "``-title-icon``" keys of the various admonitions to tell Sphinx to use arbitrary code: for example there is a ``fontawesome`` package which can be loaded in preamble and then one can use its commands, for example `div.note_title-icon=\faInfoCircle`.

Arrgh, I discover one can not load package `fontawesome` if package `fontawesome5` is already loaded... we need to think of a way to handle that because currently the code loads automatically `fontawesome5` if available.

It seems unavailable to add some conf.py variable here alas.